HerPublicWebsite.ManagementShell/CommandHandler.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace HerPublicWebsite.ManagementShell;
+
+public class CommandHandler
+{
+    private readonly IDatabaseOperation databaseOperation;
+    private readonly IFakeReferralGenerator fakeReferralGenerator;
+    private readonly IOutputProvider outputProvider;
+
+    public CommandHandler(IDatabaseOperation databaseOperation, IFakeReferralGenerator fakeReferralGenerator,
+        IOutputProvider outputProvider)
+    {
+        this.databaseOperation = databaseOperation;
+        this.fakeReferralGenerator = fakeReferralGenerator;
+        this.outputProvider = outputProvider;
+    }
+
+    public void GenerateReferrals(string[] args)
+    {
+        if (args.Length == 0)
+        {
+            outputProvider.Output("Missing number of referrals. Usage: 'GenerateReferrals <count>'.");
+            return;
+        }
+
+        if (!int.TryParse(args[0], out var referralCount))
+            outputProvider.Output("Invalid number of referrals. Usage: 'GenerateReferrals <count>'.");
+
+        outputProvider.Output("!!!!!!!!!!!!!!!!!!!!!!");
+        outputProvider.Output(
+            $"You are about to generate {referralCount} fake referral requests and add them to the database.");
+        outputProvider.Output("This command should only be run on development and staging environments.");
+        outputProvider.Output("Double check that you are NOT running this command on a production environment.");
+        outputProvider.Output("!!!!!!!!!!!!!!!!!!!!!!");
```

Comment:
Can we do something to make rollback easy in case (god forbid) this ever got run on prod? Something like having all the addresses start with something obvious like `FAKEDATA:`? Such that we could manually run a SQL DELETE operation safely.

Reply:
have added FAKE USER to the start of their name

Reply:
Could you document that somewhere alongside this (maybe in the script output at least?)?

Reply:
have added a note here